.dockerignore

```diff
@@ -1 +1,31 @@
-node_modules
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file for more about ignoring files.
```

Comment:
should we ignore `.env` as well?

Reply:
I wasn't thinking on the image size, but that it could be a security problem to have the .env bundled, similar to why

```ruby
# Ignore all default key files
config/master.key
config/credentials/*.key
```

are ignored I guess

Reply:
I tried to ignore all the `.env` files except for `.env.test`. Let me know what you think now 🙂

Reply:
great 💪